package.json

```diff
@@ -26,5 +26,9 @@
     "ts-node": "^9.1.1",
     "typescript": "^4.2.4",
     "vite": "^2.2.3"
+  },
+  "dependencies": {
+    "@types/inquirer": "^7.3.1",
```

Comment:
`@types/inquirer` is a devDependency.
You can uninstall and reinstall it:
`npm uninstall @types/inquirer`
`npm i -D @types/inquirer`